src/contributing/index.md

```diff
@@ -1,13 +1,36 @@
 # Contributing
 
-There's more to running a distribution than just writing code. This section
-explains how to be an active part of Void.
+There's more to running a distribution than just writing code.
 
-Please also visit the Void Web site for further information about [how to
-participate](https://voidlinux.org/contribute/), including our communication
-channels and how to contribute to the Void package repository.
+To contribute to the Void packages repository, start by reading the
+[CONTRIBUTING](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)
+document in the void-packages GitHub repository.
 
-## Section Contents
+To contribute to the Void Handbook, read
+[CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md)
+in the void-docs GitHub repository.
```

Comment:
"To contribute to this Handbook, read CONTRIBUTING in the void-docs repository" ?

Not mention GitHub on both to avoid repetition? I would go as far as not mentioning GH at all in both paragraphs.

Reply:
i've locally changed the second para to your suggested text.

i have a slight preference to retaining 'GitHub' in the first para, though, due to how often people conflate the `void-packages` GH repo with the binary package repo(s). But if you and @bobertlo would both prefer i remove it, i'm happy to do so.

Reply:
For the first para, you have a good point. "package repository" and "git repository" can be confusing for new comers.

Reply:
I have no strong opinion. I think it's fine leaving it.

Reply:
Okay, leaving it - thanks.